packages/adblocker-electron/adblocker.ts

```diff
@@ -23,25 +24,21 @@ export function fromElectronDetails({
   url,
   resourceType,
   referrer,
-}: {
-  url: string;
-  resourceType: ElectronRequestType;
-  referrer: string;
-}): Request {
+}: Electron.OnBeforeRequestDetails): Request {
   return Request.fromRawDetails({
     sourceUrl: referrer,
-    type: resourceType || 'other',
+    type: (resourceType || 'other') as ElectronRequestType,
     url,
   });
 }
 
 /**
  * Wrap `FiltersEngine` into a Electron-friendly helper class.
  */
-export default class ElectronBlocker extends Engine {
+export class ElectronBlocker extends FiltersEngine {
   public enableBlockingInSession(ses: Electron.Session) {
     ses.webRequest.onBeforeRequest({ urls: ['<all_urls>'] }, this.onRequest);
-    ses.setPreloads(['./dist/content.js']);
+    ses.setPreloads([join(__dirname, './content.js')]);
```

Comment:
@sentialx I'm not sure about how to get the right path here. Maybe I broke something.